week04/139. Word Break.cpp

```diff
@@ -0,0 +1,27 @@
+// https://leetcode.com/problems/word-break/
+// Time Complexity: O(N*M*K)
+// Spcae Complexity: O(N)
```

Comment:
I see what you did. It is a good solution but not optimal. There is a high chance interviewer will reject this one.